part_2_dynamic.md

```diff
@@ -4,66 +4,95 @@ title: Part 2
 permalink: /part_2/
 ---
 
-# Part 2 - Creating the Rails app
+# Part 2 - Introducing Rails
 
-* Open the terminal and create a new Rails app
+## Creating your (first) Rails app
+
+Open the terminal to create a new Rails app
 
 {% highlight bash %}
-rails new railsgirls-app
-cd railsgirls-app
+rails new railsgirls-blog
+cd railsgirls-blog
 {% endhighlight %}
 
-* Have a look how Rails generated a folder structure and a lot of files for you. There are two ways of doing that, in the terminal or with Finder/Explorer. Ask your coach about the details. Once you have seen them, move your ```about.html``` to the public folder.
+Find the project directory in the Finder/Explorer. Have a look how Rails generated a folder structure and a lot of files for you. Ask your coach about the details.
+
+
+## Starting the Rails server
+
+A Rails app can deliver dynamically generated content with the help of the Ruby programming language. This is why we can't just open a file anymore in the browser. We need a Rails server that will allow for all the magic to happen.
+
+You start your Rails server as follows. Open you terminal (in the `railsgirls-blog` folder) and type the following:
 
 {% highlight bash %}
-mv --your-downloads-folder-here--/about.html public/
-(i.e. mv ~/Downloads/about.html public/)
+rails server
 {% endhighlight %}
 
-## Start rails app
+Once it's started up, you can go visit your app already under
 
-* `rails server`
-* visit [http://localhost:3000/about.html](http://localhost:3000/about.html)
+[http://localhost:3000/](http://localhost:3000/).
 
-# Create pages controller
+It's not much but the page indicates that everything worked fine until now and the server is running.
```

Comment:
~~It's not much but~~ The page indicates that everything worked fine ~~until now~~ and the server is running